bundles/core/org.eclipse.smarthome.core/ESH-INF/config/networkinterface.xml

```diff
@@ -10,6 +10,12 @@
 			<description>A subnet (e.g. 192.168.1.0/24) <!-- or an IP address (e.g. 192.168.1.5) --></description>
 			<limitToOptions>true</limitToOptions>
 		</parameter>
+        <parameter name="broadcastAddress" type="text">
```

Comment:
Have you applied the formatter (remember the Eclipse IDE does not support save actions and so auto formatting on save for XML files)?
Shouldn't XML files be indented by tabs?

Reply:
Done.